dlt/common/destination/reference.py

```diff
@@ -577,14 +579,19 @@ def with_staging_dataset(self) -> ContextManager["JobClientBase"]:
 class SupportsStagingDestination:
     """Adds capability to support a staging destination for the load"""
 
+    def __init__(self, config: DestinationClientDwhWithStagingConfiguration) -> None:
```

Comment:
this should be removed. `SupportsStagingDestination` is a mixing class, should not hold any state.

Reply:
done